WordPress/Classes/Services/SharingService.swift

```diff
@@ -354,14 +361,16 @@ open class SharingService: LocalCoreDataService {
     ///
     /// - Returns: A `PublicizeConnection`.
     ///
-    fileprivate func createOrReplacePublicizeConnectionForBlogWithObjectID(_ blogObjectID: NSManagedObjectID,
-        remoteConnection: RemotePublicizeConnection) throws -> PublicizeConnection {
-
-            let blog = try managedObjectContext.existingObject(with: blogObjectID) as! Blog
-            let pubConn = PublicizeConnection.createOrReplace(from: remoteConnection, in: self.managedObjectContext)
-            pubConn.blog = blog
+    fileprivate func createOrReplacePublicizeConnectionForBlogWithObjectID(
```

Comment:
Could this be `private`?

```suggestion
    private func createOrReplacePublicizeConnectionForBlogWithObjectID(
```

Reply:
Updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/20265/commits/27c44c77c841cec3a2467b6ce7c4dcd18bcd9cd3